build_vs2017/ssc_vs2017.vcxproj

```diff
@@ -44,7 +44,6 @@
     <ClCompile Include="..\ssc\cmod_merchantplant.cpp" />
     <ClCompile Include="..\ssc\cmod_merchantplant_eqns.cpp" />
     <ClCompile Include="..\ssc\cmod_mhk_costs.cpp" />
-    <ClCompile Include="..\ssc\cmod_mhk_eqns.cpp" />
```

Comment:
build_vs2017 eliminated but cmod_mhk_eqns should remain

Reply:
That was the bug I'd introduced!